includes/Client.hpp

```diff
@@ -43,7 +43,8 @@ class Client {
   bool _realnameSet;
   bool _passwordGiven;
   bool _accepted;
-  UserModes _uModes;
+  bool _CapSend;
```

Comment:
I suppressed this on my last pull request. Did you merge the main properly ? Otherwise we should delete this.

Reply:
Removed this line !